amundsen_application/static/js/components/TableDetail/ColumnListItem/index.spec.tsx

```diff
@@ -132,7 +132,113 @@ describe('ColumnListItem', () => {
       expect(newWrapper.find(ColumnStats).exists()).toBe(true);
     });
 
-    it('renders EditableSection with correct edit text and Url when expanded', () => {
+    it('renders EditableSection with correct description, edit text and Url when expanded', () => {
+      instance.setState({ isExpanded: true });
+      const newWrapper = shallow(instance.render());
+      expect(newWrapper.find(EditableSection).exists()).toBe(true);
+      const editableSection = newWrapper.find(EditableSection);
+      expect(editableSection.props()).toMatchObject({
+        title: 'Description',
+        readOnly: !props.data.is_editable,
+        editText: props.editText,
+        editUrl: props.editUrl,
+      });
+    });
+
+    it('renders EditableSection with non-empty description, empty edit text and url for editable column description when expanded', () => {
+      const { props, wrapper } = setup({
+        editText: '',
+        editUrl: '',
+      });
+      const instance = wrapper.instance();
+      instance.setState({ isExpanded: true });
+      const newWrapper = shallow(instance.render());
+      expect(newWrapper.find(EditableSection).exists()).toBe(true);
+      const editableSection = newWrapper.find(EditableSection);
+      expect(editableSection.props()).toMatchObject({
+        title: 'Description',
+        readOnly: !props.data.is_editable,
+        editText: props.editText,
+        editUrl: props.editUrl,
+      });
+    });
+
+    it('renders EditableSection with empty description, edit text and url for editable column description when expanded', () => {
```

Comment:
Nit: I would nest this test and the next within a describe clause that says `describe('when empty description', ...`